### PR TITLE
link to the docs source file in Github on each docs page

### LINKED ISSUE #104: Add a button/link that lets you easily file issues on the website repo
## What happened?

* What were you attempting to do?

I saw a typo on a Paketo docs page and I wanted to quickly open an issue or pull request calling out the problem.

* What did you expect to happen?
I expected to see

a) an "edit" button that would drop me into the Github UI for proposing an edit to the page (e.g. https://github.com/paketo-buildpacks/paketo-website/edit/main/content/docs/builders/_index.md), and/or

b) a link/button that brings me to the [Paketo site issues](https://github.com/paketo-buildpacks/paketo-website/issues) so I can easily file an issue re: the docs.

* What was the actual behavior? Please provide log output, if possible.
I had to figure out where the site lives in Github, dig through the markdown files for the one where I saw the error, and then open an issue with my suggestion.

## Build Configuration

* What platform (`pack`, `kpack`, `tekton` buildpacks plugin, etc.) are you
  using? Please include a version.

* What buildpacks are you using? Please include versions.

* What builder are you using? If custom, can you provide the output from `pack
  inspect-builder <builder>`?

* Can you provide a sample app or relevant configuration (`buildpack.yml`,
  `nginx.conf`, etc.)?

## Checklist

* [ ] I have included log output.
* [ ] The log output includes an error message.
* [ ] I have included steps for reproduction.

### PR DESCRIPTION
**Pull request by fg-j**
## Summary

Resolves #104 . It's not the prettiest thing, but it is functional. Happy to field suggestions on a better UX.

## Checklist

* [x] I have viewed, signed, and submitted the Contributor License Agreement.
* [x] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [ ] I have added an integration test, if necessary.
* [ ] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.

### CODE REVIEW

**Comment by ForestEckhardt** [comment]
I think that I am going to defer on this one to @kvedurmu. I think that a button would look nicer and be less intrusive. But I am not married to that idea.

**Comment by kvedurmu** [comment]
Yeah. I agree with @ForestEckhardt that we should include a button called "Edit" and link straight to the edit page for the individual docs. The buttons on buildpacks.io (e.g https://buildpacks.io/docs/) look really good, and maybe we can re-use that and just style it to paketo colors.

**Comment by fg-j** [comment]
@kvedurmu I gave it a shot. Let me know what you think. I rebased on Forest's changes so it's easier to get a sense of what it'd really look like. @ForestEckhardt it seems like you have a bit more CSS finesse. Maybe There's a better way to do this that I just didn't figure out.

**Comment by kvedurmu** [comment]
This button looks great @fg-j. As I mentioned in https://github.com/paketo-buildpacks/paketo-website/pull/110#issuecomment-786777784, I think we should get a designer to look at the updated header, so it could be simpler to separate this from Forest's PR.

**Comment by fg-j** [comment]
@kvedurmu I'll wait until @ForestEckhardt updates the PR to extract the header work. Because I still want this one to take advantage of the improved link styling and stuff.

**Comment by kvedurmu** [comment]
@fg-j Forest's PR has been merged into main, so you can go ahead and rebase off of that.

**Review by kvedurmu — CHANGES_REQUESTED** [review]
It looks like the "Edit" text is not lined up with the Github icon. Another suggestion I'd make is to create a box around this so that it stands out more on the page. See this [buildpacks.io example](https://buildpacks.io/docs/) for reference
 
image

File: layouts/partials/docs-body.html [hunk 1]
```diff
@@ -4,9 +4,16 @@
     </div>
     <div class="docs-body relative">
         <div class="text-col docs-text pad50">
-            {{ .Content }}
+          <div class="upper-nav right">
+            <div class="edit-button">
+              <a href="http://github.com/paketo-buildpacks/paketo-website/edit/main/content/{{ .File.Path }}" target="_blank" class=edit></a>
```
Comment (kvedurmu) [inline layouts/partials/docs-body.html:9]: Is it possible to also add this button to the bottom of the page?

**Comment by fg-j** [comment]
Closing because this is addressed in #111

**Pull request closed without merging**